Gem/Code/multiplayersample_files.cmake

```diff
@@ -11,18 +11,6 @@ set(FILES
     # Scripting samples
     Source/AutoGen/ScriptingPlayerMovementComponent.AutoComponent.xml
```

Comment:
Can this move to _autogen_files.cmake as well? It would be good to have all the XMLs in one place.

Reply:
Might as well now that I've removed the AutoGen target.